Header  header

int16   gnss_week
float64 gnss_time

float64 heading
float64 pitch
float64 roll

float64 latitude
float64 longitude
float64 altitude

float64 velocity_e
float64 velocity_n
float64 velocity_u

float64 baseline
int16   nsv1
int16   nsv2

char[2] status